test/input/Comment-Class.cxx

```diff
@@ -0,0 +1,2 @@
+/** class omment */
```

Comment:
We've found that (our usage of) `libclang` does not enjoy parsing non-docstring comments that precede docstrinsgs, e.g.:
```cc
// TODO: Fixed docstrings.
/** this comment gets parsed, yay! */
class A;

/** this comment does not get parsed, dern! */
// TODO: Screw up docstrings.
class B;
```

Can that behavior be tested somewhere in here?

Reply:
FTR Here is our usage:
https://github.com/RobotLocomotion/drake/blob/74292cacd1c42d6b3e682dc836254cdb834ea2e6/third_party/com_github_pybind_pybind11/mkdoc.py#L164-L225

Reply:
I updated the test suite to cover

```c++
// Non-doc comment before.
/** doc comment */
// Non-doc comment after.
void start(int);
```

CastXML correctly reports the doc comment.

Reply:
This isn't actually relevant to the test case passing or failing, but I guess this comment was meant to read "class comment" instead of "class omment"?

Reply:
Fixed, thanks.